exonum/src/runtime/dispatcher/tests.rs

```diff
@@ -228,9 +227,8 @@ impl Runtime for SampleRuntime {
                 self.services.insert(spec.id, new_status);
                 self.new_services.insert(spec.id, new_status);
             }
-            Ok(())
         } else {
-            Err(DispatcherError::IncorrectRuntime.into())
+            panic!(DispatcherError::IncorrectRuntime)
```

Comment:
Nit: Does panicking with an error (instead of a string) make sense here? Is the error extracted via `catch_unwind`? Otherwise, it could make sense to panic with a string, IMO.